README.md

```diff
@@ -76,9 +76,12 @@ In the examples, intentions are created in the Ansible playboook [00_create_inte
 | hive\_container\_port | Hive container port | number | 9083 | no |
 | hive\_docker\_image | Hive container image | string | "fredrikhgrelland/hive:3.1.0" | no |
 | hive\_container\_environment\_variables | Hive environment variables | list(string) | [""] | no |
-| resource | Resource allocations | object |  | no |
-| resource.cpu | Resource allocation - cpu | number | 500 | no |
-| resource.memory | Resource allocation - memory | number | 1024 | no |
+| resource | Resource allocations | object | - | no |
+| resource.cpu | Resource allocation - cpu | number | 100 | no |
+| resource.memory | Resource allocation - memory | number | 128 | no |
```

Comment:
```suggestion
| resource.cpu | Resource allocation - cpu | number | 500 | no |
| resource.memory | Resource allocation - memory | number | 1024 | no |
```

These are set as default hive resources now to 500, 1024. Should the `README` be updated or the resource in `variables.tf` ?

Reply:
It seems that the default value for `cpu` is 250. So I guess `cpu = 500` is correct?

Reply:
Yes, `cpu=500`  and `memory=1024` as default, and for proxy it's `cpu=200`  and `memory=128`